### PR TITLE
searxng: unstable-2023-10-31 -> 0-unstable-2024-02-24

### PR DESCRIPTION
**Pull request by SuperSandro2000**
Diff: https://github.com/searxng/searxng/compare/b05a15540e1dc2dfb8e4e25aa537b2a68e713844..c7309a0869da99856ccb101b4a37a71087b863bb

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/se/searxng/package.nix [hunk 1]
```diff
@@ -5,13 +5,13 @@
 
 python3.pkgs.toPythonModule (python3.pkgs.buildPythonApplication rec {
   pname = "searxng";
-  version = "unstable-2023-10-31";
+  version = "unstable-2024-02-23";
```
Comment (999eagle) [inline pkgs/by-name/se/searxng/package.nix:8]: Should we change this to be `1.0.0-unstable-2024-02-23` to conform to the new versioning scheme?

Comment (SuperSandro2000) [reply pkgs/by-name/se/searxng/package.nix:8]: I don't know, they changed the versioning scheme since and are not really tagging things anymore.

Comment (999eagle) [reply pkgs/by-name/se/searxng/package.nix:8]: I guess then we should use `0-unstable-2024-02-23`

Comment (SuperSandro2000) [reply pkgs/by-name/se/searxng/package.nix:8]:
```suggestion
  version = "0-unstable-2024-02-23";
```

Comment (SuperSandro2000) [reply pkgs/by-name/se/searxng/package.nix]: Doesn't work with setuptools 

```
setuptools.extern.packaging.version.InvalidVersion: Invalid version: 'unstable.2024.02.24+d72fa99b'
```

**Review by 999eagle — APPROVED** [review]
Tested and deployed to my instance, LGTM

**Comment by Mellowchan** [comment]
This update was not tested properly. 
The service is trying to start _searx-run_  and it fails but the actual name is of the executable is _searxng-run_

**Comment by SuperSandro2000** [comment]
> https://github.com/NixOS/nixpkgs/pull/290962

It was but I think the script is not relevant when using uswgi.

**Comment by SuperSandro2000** [comment]
Please test #292157

**Pull request merged**